assets/templates/oauth_clients_limit_exceeded.html

```diff
@@ -22,7 +22,8 @@
           <div class="modal-body mt-4 mt-md-1 p-md-5">
             <h1 class="h4 h2-md mb-4 text-center">{{t "Too Many OAuth Clients Title" .ClientsCount .ClientsLimit}}</h1>
             <p class="mb-4 text-center">
-              {{t "Too Many OAuth Clients Intro" .ClientsLimit}}
+              {{t "Too Many OAuth Clients Intro" .ClientsLimit}}&nbsp;-
```

Comment:
[nit] I don't think there is a space before `;` in english

Reply:
the `;` is part of the non-breakable space character `&nbsp;`, it's not part of the text.